pennylane/ops/op_math/controlled_decompositions.py

```diff
@@ -190,53 +274,24 @@ def decomp_circuit(op):
             f"got {target_operation.__class__.__name__}."
         )
     control_wires = Wires(control_wires)
-    if len(control_wires) > 1:
-        raise ValueError(
-            f"The control_wires should be a single wire, instead got: {len(control_wires)} wires."
-        )
 
     target_wire = target_operation.wires
 
     if isinstance(target_operation, Operation):
         try:
-            phi, theta, omega = target_operation.single_qubit_rot_angles()
+            rot_angles = target_operation.single_qubit_rot_angles()
```

Comment:
A bit of a side note here, we are trying to not use `target_operation.single_qubit_rot_angles()` anymore since it was supposed to be deprecated. However, it turned out to be more cumbersome than expected. In [the branch I was working on this](https://github.com/PennyLaneAI/pennylane/compare/master...deprecate-single-rot-angles), I changed these lines to:

```python
phi, theta, omega, global_phase = _single_qubit_rot_angles_from_matrix(
        qml.matrix(target_operation)
    )
```
Not a blocking comment since this turned out to require way more work than just changing this line.